src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -654,6 +667,76 @@
     text: az sql midb short-term-retention-policy show -g mygroup --mi myinstance -n mymanageddb --deleted-time "2018-05-20T05:34:22"
 """
 
+helps['sql midb long-term-retention-policy'] = """
```

Comment:
`long-term-retention-policy` may be too long for users to type in command line. Can we consider abbrevation for this?

Reply:
yep, planning to shorten

Reply:
done